daprdocs/content/en/reference/components-reference/supported-configuration-stores/azure-appconfig-configuration-store.md

```diff
@@ -0,0 +1,86 @@
+  - name: azureEnvironment # Optional, defaults to AZUREPUBLICCLOUD
+    value: "AZUREPUBLICCLOUD"
+  # See authentication section below for all options
+  - name: azureTenantId # Optional
+    value: "[your_service_principal_tenant_id]"
+  - name: azureClientId # Optional
+    value: "[your_service_principal_app_id]"
+  - name: azureCertificateFile # Optional
+    value : "[pfx_certificate_file_fully_qualified_local_path]"
+
+```
+
+{{% alert title="Warning" color="warning" %}}
+The above example uses secrets as plain strings. It is recommended to use a secret store for the secrets as described [here]({{< ref component-secrets.md >}}).
+{{% /alert %}}
+
+## Spec metadata fields
+
+| Field             | Required | Details | Example |
+|-------------------|:--------:|---------|---------|
+| host              | N        | Endpoint for the Azure App Configuration instance. No Default  | `https://dapr.azconfig.io`
+| connectionString  | N        | Connection String the Azure App Configuration instance. No Default. Can be `secretKeyRef` to use a secret reference  | `Endpoint=https://foo.azconfig.io;Id=osOX-l9-s0:sig;Secret=00000000000000000000000000000000000000000000`
+| maxRetries        | N         | Maximum number of retries before giving up. Defaults to `3` | `5`, `10`
+| retryDelay        | N         | RetryDelay specifies the initial amount of delay to use before retrying an operation. The delay increases exponentially with each retry up to the maximum specified by MaxRetryDelay. Defaults to `4` seconds; `"-1"` disables delay between retries. | `4000000000`
+| maxRetryDelay     | N         | MaxRetryDelay specifies the maximum delay allowed before retrying an operation. Typically the value is greater than or equal to the value specified in RetryDelay. Defaults to `120` seconds; `"-1"` disables the limit | `120000000000`
+
+**Note**: either `host` or `connectionString` must be specified.
+
+## Authenticating with Connection String 
+
+To access an App Configuration instance, you can use its connection string, which is available in the Azure portal. Because connection strings contain credential information, it is recommended treat them as secrets and use a secret store as described [here]({{< ref component-secrets.md >}})
+
+## Authenticating with Azure AD
+
+The Azure App Configuration configuration store component also supports authentication with Azure AD. Before you enable this component, make sure you've read the [Authenticating to Azure]({{< ref authenticating-azure.md >}}) document and created an Azure AD application (also called Service Principal). Alternatively, make sure you have created a managed identity for your application platform.
+
+## Setup Azure App Configuration
+
+**Note**: An Azure Subscription is requiered.
+
+1. [Start the Azure App Configuration creation flow](https://ms.portal.azure.com/#create/Microsoft.Azconfig). Log in if necessary.
+2. Click "Create" to kickoff deployment of your Azure App Configuration instance.
```

Comment:
```suggestion
1. Click **Create** to kickoff deployment of your Azure App Configuration instance.
```

Changing the numbered list to all "1" will automatically order the list correctly for you.